docs/intro/tutorial02.txt

```diff
@@ -473,8 +473,14 @@ Save these changes and start a new Python interactive shell by running
     >>> from polls.models import Choice, Question
 
     # Make sure our __str__() addition worked.
-    >>> Question.objects.all()
+    >>> q = Question.objects.all()
+    >>> q
     <QuerySet [<Question: What's up?>]>
+    # Print the SQL used to query the database
+    >>> from pprint import pprint
+    >>> pprint(str(q.query))
+    ('SELECT "polls_question"."id", "polls_question"."question_text", '
+    '"polls_question"."pub_date" FROM "polls_question"')
```

Comment:
Hi thanks for the suggestion ♥️

In this particular example the `pprint(str(query))` may be unnecessary as `print(query)` is enough.

Reply:
Thanks for the comments! I removed both `str` calls.